packages/dso-toolkit/src/components/document-list/document-list.scss

```diff
@@ -9,6 +9,10 @@
   @include list.unstyled();
 
   margin: 0 css-grid-variables.$grid-gutter-width * -0.5 0 css-grid-variables.$grid-gutter-width * -0.5;
```

Comment:
Mijn scss is wat roestig als het om dit soort dingen gaat, maar is er misschien een manier voor ons om de berekeningen tussen haakjes te zetten voor de leesbaarheid? Zoals iets als dit?

```
margin: 0 (css-grid-variables.$grid-gutter-width * -0.5) 0 (css-grid-variables.$grid-gutter-width * -0.5);
```

Of om juist dit soort dingen uit te schrijven als
```
margin-block: 0; 
margin-inline-start: $grid-gutter-width * -0.5;
```

Reply:
Good catch! 👍